.github/workflows/ci.yml

```
@@ -29,6 +29,17 @@ jobs:
           command: check
           args: --workspace --all-targets --all-features
 
+  check_no_std:
+    name: Check no_std
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: dtolnay/rust-toolchain@1.64.0
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
+          args: --package=ash --no-default-features --features=debug,linked
```

To match the format of the rest of this file:

```suggestion
          args: -p ash --no-default-features --features=debug,linked
```